secret_non_test.rego

```diff
@@ -1,12 +1,13 @@
-package policy
+# METADATA
+# title: Minimal Code
```

Comment:
This title doesn't match the rest of the file.

Reply:
Updated